cybersixgill_actionable_alerts/assets/configuration/spec.yaml

```
@@ -0,0 +1,58 @@
+name: cybersixgill_actionable_alerts
+files:
+- name: cybersixgill_actionable_alerts.yaml
+  options:
+  - template: init_config
+    options:
+    - template: init_config/default
+  - template: instances
+    options:
+      - name: cl_id
+        required: true
+        description: The Client Id given by Cybersixgill
+        enabled:  true
+        value:
+          type: string
+          example:  clientid
+          display_default:  null
+      - name: cl_secret
+        required: true
+        description:  The Client Secret given by Cybersixgill
+        enabled:  true
+        value:
+          type: string
+          example:  client00000secret
+          display_default:  null
+      - name: alerts_limit
+        required: false
+        description:  The number of alerts to fetch on a single request default is 50
+        enabled:  false
+        value:
+          type: string
+          example:  "50"
```

```suggestion
          type: string
          example:  "50"
```
```suggestion
          type: integer
          example:  50
```
Can set this as an integer instead.